machines/get-connection.js

```diff
@@ -148,24 +167,44 @@ module.exports = {
       client.on('error', function onIntraConnectionError (err){
         // If manager was not provisioned with an `onUnexpectedFailure`,
         // we'll just handle this error event silently (to prevent crashing).
-        if (!isFunction(inputs.manager.onUnexpectedFailure)) {
+        if (!_.isFunction(inputs.manager.onUnexpectedFailure)) {
```

Comment:
@sgress454 side note for future reference: I could have just done this as a truthiness check in the first place since we already have the guarantee from rttc.  Not worth messing with or anything, just leaving this message for posterity's sake